src/player.cpp

```diff
@@ -3800,10 +3768,10 @@ void player::disp_status(WINDOW *w, WINDOW *w2)
     int y  = sideStyle ?  0 :  3;
     int dx = sideStyle ?  0 :  6;
     int dy = sideStyle ?  1 :  0;
-    mvwprintz(w, y + dy * 0, x + dx * 0, col_str, _("Str%2d"), get_str());
-    mvwprintz(w, y + dy * 1, x + dx * 1, col_dex, _("Dex%2d"), get_dex());
-    mvwprintz(w, y + dy * 2, x + dx * 2, col_int, _("Int%2d"), get_int());
-    mvwprintz(w, y + dy * 3, x + dx * 3, col_per, _("Per%2d"), get_per());
+    mvwprintz( w, y + dy * 0, x + dx * 0, col_str, _("Str%2d"), str_cur );
```

Comment:
It obfuscated the bug. In theory the getter should return the same thing as `[stat]_cur` and thus make it OK to use those interchangeably, but it didn't.

Most calculations in the game use stat members directly, so I decided to first fix the problem rather than doing the full replace of members by getters.

Transition from members to getters should probably start by making getters return members without performing any operations on them.